superset-frontend/src/filters/components/Select/SelectFilterPlugin.tsx

```diff
@@ -72,6 +83,60 @@ function reducer(
   }
 }
 
+const StyledTag = styled(AntdTag)`
+  & .ant-tag-close-icon {
+    display: inline-flex;
+    align-items: center;
+    margin-left: ${({ theme }) => theme.gridUnit}px;
+  }
+
+  & .tag-content {
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+`;
+
+export interface CustomTagProps extends HTMLSpanElement {
+  label: ReactNode;
+}
+
+// TODO: use antd Tag props instead of any. Currently it's causing a typescript error
+const Tag = (props: any) => {
```

Comment:
It seems the custom render is necessary to make `oneLine` work correctly so we need to move this code to the Select component.

Reply:
done